modules/learn/pages/clusters-and-availability/rebalance.adoc

```diff
@@ -31,47 +39,65 @@ If and when the missing Data Service node is restored or replaced, rebalance wil
 See xref:learn:clusters-and-availability/intra-cluster-replication.adoc[Intra-Cluster Replication], for information on how data is distributed across nodes.
 
 [#data-service-rebalance-stages]
-=== Data-Service Rebalance Stages
+=== Data-Service Rebalance Substages
 
-vBuckets are moved in _stages_.
-The stages &#8212; which differ, depending on whether the vBucket is an _active_ or a _replica_ vBucket &#8212; are described below.
+During the Data Service rebalance stage, vBuckets are moved in _substages_.
```

Comment:
I would prefer the phrasing a vbucket move consists of certain phase(s). The phases depend on whether the vbucket moved is an active or replica vbucket move. 

data-service rebalance has substages example, deltaRecovery substage which signifies the the delta recovery warmup substage of dat service service rebalance.

Reply:
Okay, I've used 'phase' and 'subphase'. See what you think. I've changed rebalance.adoc and also rebalance-reference.adoc to read in this way.